L1Trigger/TrackFindingTMTT/src/Settings.cc

```diff
@@ -0,0 +1,508 @@
+#include <L1Trigger/TrackFindingTMTT/interface/Settings.h>
+#include "FWCore/Utilities/interface/Exception.h"
+#include <set>
+#include <cmath>
+
+namespace tmtt {
+
+  // Set config params for HYBRID TRACKING via hard-wired consts to allow use outside CMSSW.
+
+  Settings::Settings() {
+    //
+    // TMTT related configuration parameters, including Kalman Filter.
+    // Meaning of these parameters explained in TrackFindingTMTT/python/TMTrackProducer_Defaults_cfi.py
+    //
+    reduceLayerID_ = true;
+    useLayerID_ = true;
+    minFracMatchStubsOnReco_ = -99;
+    minFracMatchStubsOnTP_ = -99;
+    minNumMatchLayers_ = 4;
+    minNumMatchPSLayers_ = 0;
+    stubMatchStrict_ = false;
+    minStubLayers_ = 4;
+    minPtToReduceLayers_ = 99999.;
+    deadReduceLayers_ = false;
+    kalmanMinNumStubs_ = 4;
+    kalmanMaxNumStubs_ = 6;
+    numPhiNonants_ = 9;
+    numPhiSectors_ = 9;
+    etaRegions_ = {-2.4,
```

Comment:
The number of phi nonants could be obtained from the geometry. Our goal is to make this change when we interface the L1 tracking to the Tracker DTC emulation, in about a year's time. The number of phi sectors and eta regions can't, since they are configuration parameters of the L1 tracking.